### DISCUSSION TITLE
Question about menu config

### DISCUSSION THREAD

**Discussion by Astrantia** [hrsh7th/nvim-cmp#606] [Q&A] [answered]
https://github.com/neovim/neovim/pull/16499

How did you manage to get this appearance with icons to the left?

**Comment by hrsh7th** [comment, marked as answer]
https://github.com/hrsh7th/nvim-cmp#formattingfields-type-cmpitemfield
https://github.com/hrsh7th/nvim-cmp/issues/596
https://github.com/hrsh7th/nvim-cmp/issues/583
https://github.com/hrsh7th/nvim-cmp/issues/576
